src/PokemonTrainer.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class PokemonTrainer {
+    String name;
+    List<Pokemon> Pokemons;
+
+    ElectricPokemon raichu = new ElectricPokemon("Raichu", 45, 35, "thunders", "BOOOM");/*⚡*/
+    GrassPokemon venusaur = new GrassPokemon("Venusaur", 5, 70,"seeds", "grumble");/*🌿*/
+    GrassPokemon ditto = new GrassPokemon("Ditto", 35, 85, "trees", "sssssssh");/*🌿*/
+    FirePokemon charizard = new FirePokemon("Charizard", 15, 60, "lava", "rommeldebommel");/*🔥*/
+    WaterPokemon blastoise = new WaterPokemon("Blastoise", 20, 70, "hurricanes", "whoosh");/*🌊*/
+    WaterPokemon gyarados = new WaterPokemon("Gyarados", 30, 89, "cyclons", "what?");/*🌊*/
+
+    public List<Pokemon> getPokemonList() {
+        return Arrays.asList(getCharizard(), getBlastoise(), getDitto(), getGyarados(), getRaichu(), getVenusaur());
+    }
+
+//----- constructor -------------------------------------------------------------------------------------------
+    public PokemonTrainer(String name) {
+        this.name = name;
+    }
+
+    public PokemonTrainer(String name, ElectricPokemon raichu, GrassPokemon venusaur, GrassPokemon ditto, FirePokemon charizard, WaterPokemon blastoise, WaterPokemon gyarados) {
```

Comment:
Je hebt alle pokemon objecten hierboven al een waarde geven. Je hoeft ze dus niet meer in de constructor te zetten (tenzij je wil dat de waardes kunnen veranderen en dat de waardes die je al gedeclareert hebt slechts de standaard implementaties zijn)